plugins/se.infomaker.ximconcept/components/ConceptItemComponent.js

```diff
@@ -0,0 +1,127 @@
+import { Component } from 'substance'
+import { ConceptService, api } from 'writer'
+import ConceptItemImageComponent from './ConceptItemImageComponent'
+import ConceptItemIcon from './ConceptItemIconComponent'
+
+class ConceptItemComponent extends Component {
+
+    getInitialState() {
+        return { isHovered: false }
+    }
+
+    async willReceiveProps(newProps) {
+        let { item } = newProps
+
+        if (item) {
+            this.setState({
+                item: await ConceptService.fetchConceptItemProperties(item)
+            })
+        }
+    }
+
+    didMount() {
+        this.Tooltip = api.ui.getComponent('tooltip')
+    }
+
+    onMouseEnter() {
+        if (!this.state.isHovered) {
+            this.refs.tooltip.extendProps({
+                show: true
+            })
+            this.extendState({ isHovered: true })
+        }
+    }
+
+    onMouseLeave() {
+        if (this.state.isHovered) {
+            this.refs.tooltip.extendProps({
+                show: false
+            })
+            this.extendState({ isHovered: false })
+        }
+    }
+
+    render($$){
+        const { item, isHovered } = this.state
+        const { propertyMap } = this.props
+        const type = (item && item[propertyMap.ConceptImTypeFull]) ? item[propertyMap.ConceptImTypeFull] : ''
+        const editable = this.props.editable ? 'editable' : ''
+        const el = $$('div')
+        
+        let icon
+        let removeIcon
+        let image
+
+        if (item) {
+            const broaderString = ConceptService.extractBroaderText(item)
+            const tooltip = $$(this.Tooltip, {
+                title: `${broaderString}`,
+                text: `${this.hasValidUUid() ? item[propertyMap.ConceptDefinitionShort] || ' - ' : 
+                    this.getLabel('invalid.uuid.label')} ${item[propertyMap.ConceptStatus] ? '(' + item[propertyMap.ConceptStatus] + ')': ''}`,
+                fixed: true,
+                parent: this,
+            }).ref('tooltip')
+
+            if (item.image && !isHovered) {
+                image = $$(ConceptItemImageComponent, {
+                    propertyMap,
+                    src: image
```

Comment:
this looks confusing, assign new element to image,  then send image as src-property.